change/@ni-nimble-components-6c68cffe-b8b5-4e15-9c84-0d429391e1e3.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "Spec update to support @mention in rich text components",
```

Comment:
Actually, I have updated for viewer as well. Since the API is common for both the editor and viewer I thought it was enough to add it in one place. However, it is better to explain it to viewer too so I just linked the same editor API sections to the viewer for better readability.

I already covered the details of how to use the configuration and mapping elements added an example in the design section and even mentioned the client guidance on how to send the mapping elements.